src/data/csvtojson.ts

```diff
@@ -43,16 +45,16 @@ fs.createReadStream('./routes.csv')
       uiaa: data.UIAA,
       ewbank: data.Ewbank,
       saxon: data.Saxon,
-      band: data['Level Bands']
+      band: 'unknown'
     })
   })
   .on('end', () => {
     const data = JSON.stringify(routeGrades)
     fs.writeFileSync('routes.json', data)
-  // [
-  //  { score: 26, yds: '5.10a', french: '3a+',saxon: '3', band: 'beginner' },
-  //  { score: 27, yds: '5.10a', french: '3b',saxon: '3', band: 'beginner' },
-  // ]
+    // [
+    //  { score: 26, yds: '5.10a', french: '3a+',saxon: '3', band: 'beginner' },
+    //  { score: 27, yds: '5.10a', french: '3b',saxon: '3', band: 'beginner' },
```

Comment:
This should this comment now have `band: unknown`?

Reply:
More generally: is this comment needed, or should we move it to a test?  It's already outdated; missing ewbank and uiaa.

Reply:
Great question. I think these comments should be removed from the file and add it to tests. I should probably run it to update the `json` files as well.

Reply:
Let's just remove the comment here and create a separate issue for tests of the parser.